rmgpy/rmg/main.py

```diff
@@ -327,13 +329,29 @@ def loadDatabase(self):
             #frequenciesLibraries = self.statmechLibraries,
             depository = False, # Don't bother loading the depository information, as we don't use it
         )
-        
+
+        # Turn off reversibility for families with three products if desired
+        if not self.trimolecularProductReversible:
+            for family in self.database.kinetics.families.itervalues():
+                if len(family.forwardTemplate.products) > 2:
+                    family.reversible = False
+                    family.reverseTemplate = None
+                    family.reverseRecipe = None
+                    family.reverse = None
+
         # Determine if trimolecular families are present
         for family in self.database.kinetics.families.itervalues():
-            if len(family.forwardTemplate.reactants) > 2 or len(family.forwardTemplate.products) > 2:
+            if len(family.forwardTemplate.reactants) > 2:
```

Comment:
Does this assume that we won't have any families with 2 reactants going to 3 products?

Reply:
No, we can have families with 2 reactants to 3 products, but trimolecular will then only be turned on if `trimolecularProductReversible` is `True`.